<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Information System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FCF7F8;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>591</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addStudent">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(85, 207, 8);
color: white;</string>
         </property>
         <property name="text">
          <string>Add Student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editStudent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 150, 30);
color: white</string>
         </property>
         <property name="text">
          <string>Edit Student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteStudent">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 80, 64);
color: white</string>
         </property>
         <property name="text">
          <string>Delete Student</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Student List:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">ID Number</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Age</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Gender</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Year Level</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Course</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>271</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="addCourse">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(88, 95, 227);
color: white</string>
         </property>
         <property name="text">
          <string>Add Course</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteCourse">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 80, 64);
color: white</string>
         </property>
         <property name="text">
          <string>Delete Course</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Courses:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget_2">
       <column>
        <property name="text">
         <string notr="true">Course Code</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Course Name</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="text">
     <string>Student Information System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchStudentField">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Student ID  or Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Student</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchCourseField">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Course Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Course</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
